tests/features/conftest.py

```diff
@@ -290,6 +290,7 @@ def value(self) -> int:
     Key('^_', b'^_', '\x1f'),
     Key('^\\', b'^\\', '\x1c'),
     Key('!resize', b'KEY_RESIZE', curses.KEY_RESIZE),
+    Key('^D', b'KEY_DC', '\x04'),
```

Comment:
ah yep the third value is correct, but the middle value should be ^D also

Reply:
good point, now done.